dasharo-compatibility/cpu-status.robot

```diff
@@ -21,10 +22,7 @@ Suite Setup         Run Keywords
 ...                     AND
 ...                     Skip If    not ${CPU_TESTS_SUPPORT}    CPU tests not supported
 ...                     AND
-...                     Power On    AND
```

Comment:
Seperated resetting the options to lib/options to make it work when ssh & dcu is used